src/Application/Application.cpp

```
@@ -90,18 +90,40 @@ void tryOpenConsole() {
 int main(int argc, char *argv[]) {
     int processStat = processAppConfig(argc, argv);
     if (processStat != EXIT_SUCCESS) return processStat;
 
     tryOpenConsole();
 
+    Engine engine;
 
     try {
         Log::BeginLogging();
         Log::PrintAppInfo();
 
```

Cppcheck (2.20.0)

[performance] Function 'getEnabledVulkanValidationLayers()' should return member 'm_enabledValidationLayers' by const reference.

(returnByReference)